src/TimeSeries.jl

```diff
@@ -9,8 +9,8 @@ using Base.Dates
 export TimeArray, AbstractTimeSeries,
        by, from, to, findwhen, findall, timestamp, values, colnames, meta,
        lag, lead, percentchange, moving, upto,
-       basecall,
-       merge, collapse,
+       basecall, uniformlyspaced, uniformlyspace,
+       merge, merge_left, collapse,
```

Comment:
Definitely, this was just a quick-and-dirty first pass - ultimately I think we should have a single merge function with various methods for inner, left, right, and outer joins.